Bottom: 8b95172447d2bd7b0c20f665546a1d306cfc89ed
Top:    1b932dd3f4087d2173953658f9464c279503c194
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-05 20:36:39 +0000

Refresh of adjust-step-export-to-2-3mm

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 610ecb5..3061489 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -77,7 +77,7 @@
 
 static Coord board_thickness;
 #define HACK_BOARD_THICKNESS board_thickness
-//#define HACK_BOARD_THICKNESS MM_TO_COORD(3.6)
+//#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 #define HACK_PLATED_BARREL_THICKNESS MM_TO_COORD(0.08)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 71d5a47..cc0b47b 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -62,7 +62,7 @@
 
 static Coord board_thickness;
 #define HACK_BOARD_THICKNESS board_thickness
-//#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+//#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 
 HID step_hid;
